Package.swift

```diff
@@ -0,0 +1,35 @@
+// swift-tools-version:5.3
```

Comment:
we can probably use a more recent version of swift-tools here. At least `5.8`, until Xcode 15 become stable.

Reply:
We should maybe consider bumping this in CocoaPods as well then, which is currently set on 5.1